.github/workflows/nmodl-ci.yml

```
@@ -0,0 +1,120 @@
+name: NMODL CI
+
+on:
+  push:
+    branches: 
+      - master
+      - live-debug*
+      - release/**
+  pull_request:
+    branches:
+      - master
+      - release/**
+
+env:
+  BUILD_TYPE: Release
+  DEVELOPER_DIR: /Applications/Xcode_11.3.1.app/Contents/Developer
+  DEFAULT_PY_VERSION: 3.8
+  DESIRED_CMAKE_VERSION: 3.8.0
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+
+    name: ${{ matrix.os }}
+
+    env:
+      INSTALL_DIR: install
+      SDK_ROOT: $(xcrun --sdk macosx --show-sdk-path)
+
+    strategy:
+      matrix:
+        os: [ ubuntu-18.04, macOS-10.15 ]
+      fail-fast: true
+
+    steps:
+
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@v1.4
+        with:
+          cmake-version: ${{ env.DESIRED_CMAKE_VERSION }}
+          
+      - name: Install homebrew packages
+        if: startsWith(matrix.os, 'macOS')
+        run: |
+          brew install coreutils bison flex boost openmpi
+        shell: bash
+
+      - name: Install apt packages
+        if: startsWith(matrix.os, 'ubuntu')
+        run: |
+          sudo apt-get install flex bison dvipng doxygen libfl-dev pandoc python3-dev python3-pip texlive-latex-recommended texlive-latex-extra
+        shell: bash
+
+      - name: Set up Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+        env:
+          PYTHON_VERSION:  ${{ env.DEFAULT_PY_VERSION }}
+       
+      - name: Install Python3 dependencies
+        run: |
+          pip3 install -U pip setuptools scikit-build Jinja2 PyYAML pytest 'sympy>=1.3,<1.6'
```

```suggestion
          pip3 install -U pip setuptools scikit-build Jinja2 PyYAML pytest 'sympy>=1.3'
```
That version requirement is gone thanks to @cattabiani 's PR some days ago.